spec/factories/refund_responses.rb

```diff
@@ -0,0 +1,16 @@
+FactoryBot.define do
+  sequence :adyen_refunds_response do
+    OpenStruct.new(
+      response: {
+        "merchantAccount" => "#{SecureRandom.uuid}",
+        "pspReference" => "#{SecureRandom.uuid}",
+        "paymentPspReference" => "#{SecureRandom.uuid}",
+        "status" => "received",
+        "amount" => {
+            "currency" => "CHF",
```

Comment:
Style/StringLiterals: Prefer single-quoted strings when you don't need string interpolation or special symbols. (https://rubystyle.guide#consistent-string-literals)